README.md

```diff
@@ -126,6 +126,13 @@ One may seed the cache using the following Rake Task:
 rake liberate:arks:seed_cache
 ```
 
+In development, it may be best to copy the cache from a production or production-like environment into `spec/fixtures/marc_to_solr/figgy_ark_cache/` and, when running traject command, set the `FIGGY_ARK_CACHE_PATH` in the local environment. e.g. 
```

Comment:
In order to index in the dev environment, we don't need to copy all the time the figgy_ark_cache from a production box. What we have right now in `spec/fixtures/marc_to_solr/figgy_ark_cache/` should work. If it doesn't work then copy from prod the `figgy_ark_cache`.

Reply:
There is only a single file in `spec/fixtures/marc_to_solr/figgy_ark_cache/` - is that what you're expecting to see there? The last time I tried with only that single file it did not seem to work - my local environment started spending a *lot* of time re-building the cache.